frontend/src/services/graphql/index.js

```diff
@@ -1,7 +1,6 @@
 export * from './Activity';
 export * from './AirflowCluster';
 export * from './ApiKeys';
-export * from './Catalog';
```

Comment:
I wonder if we want to disable catalog in the routes when Dashboards and Datasets are inactive? 
Otherwise, it will be just empty

Reply:
@jorgeig-space can you weigh-in on this?

Reply:
@dlpzx wdyt? I think ideally we should disable the Catalog if its functionality does not make sense when the other two modules that use it are inactive.

Reply:
For me `Datasets` and `Dashboards` should have a depends on on the `Catalog` module like it happens with `Datasets` and `Dataset_sharing`

Reply:
@dlpzx can you please elaborate?

Reply:
Yes, sorry I was a bit cryptic. In an ideal world, the Catalog module should be enabled if either Datasets or Dashboards are enabled. Otherwise it should be disabled. But currently it is considered `core` in the backend modularization. @nikpodsh wdyt?
option 1: keep Catalog in core (backend) or always enabled (frontend)
option 2: modularize Catalog with Dataset and Dashboards depends on Catalog (backend) and make Catalog dependent on Datasets and Dashboards (frontend)
option 3: keep Catalog in core (backend) and make Catalog dependent on Datasets and Dashboards (frontend). We will modularize Catalog (and its infrastructure) after V2.0.0 release